monai/data/wsi_reader.py

```diff
@@ -56,9 +56,11 @@ class BaseWSIReader(ImageReader):
     supported_suffixes: List[str] = []
     backend = ""
 
-    def __init__(self, level: int, **kwargs):
+    def __init__(self, level: int, channel_last: bool = False, **kwargs):
```

Comment:
I would suggest to use an arg `channel_dim: int = 0` directly instead of `channel_last`.
And add the doc-string for these 2 args.

Thanks.